app/src/main/res/layout/fragment_goal_preview.xml

```diff
@@ -47,7 +47,7 @@ Salt, to taste" />
                 android:onClick="@{() -> handler.onCreateIdeaListClick(viewModel)}"
                 android:src="@drawable/ic_create"
                 android:tint="@color/colorButton"
-                android:backgroundTint="@color/colorAccent"
+                app:backgroundTint="@color/colorAccent"
```

Comment:
@pinyaoting weren't you going to remove this feature out ? Do we need to fix this or are you going to change this anyway ?